argopy/xarray.py

```diff
@@ -727,14 +727,11 @@ def teos10(self, vlist: list = ['SA', 'CT', 'SIG0', 'N2', 'PV', 'PTEMP'], inplac
         # Coriolis
         f = gsw.f(lat)
 
-        # Depth:
-        depth = gsw.z_from_p(pres, lat)
```

Comment:
@gmaze just to be sure, you don't need to expose the calculated `depth` anywhere else, right? If so, let's merge this!

Reply:
nope, this is the only place and indeed a bug
we can nerge
g